csl-citation.json

```diff
@@ -1,101 +1,103 @@
+                "number"
+            ]
         },
         "citationItems": {
             "type": "array",
             "items": {
-                "type": "object",
-                "properties": {
-                    "id": {
-                        "type": [
-                            "string",
-                            "number" 
-                        ],
-                        "required": true 
-                    },
-                    "itemData": {
-                        "$ref": "csl-data.json/#/items" 
-                    },
-                    "prefix": {
-                        "type": "string" 
-                    },
-                    "suffix": {
-                        "type": "string" 
-                    },
-                    "locator": {
-                        "type": "string" 
-                    },
-                    "label": {
-                        "type": "string",
-                        "enum": [
-                            "book",
-                            "chapter",
-                            "column",
-                            "figure",
-                            "folio",
-                            "issue",
-                            "line",
-                            "note",
-                            "opus",
-                            "page",
-                            "paragraph",
-                            "part",
-                            "section",
-                            "sub verbo",
-                            "verse",
-                            "volume" 
-                        ] 
-                    },
-                    "suppress-author": {
-                        "type": [
-                            "string",
-                            "number",
-                            "boolean" 
-                        ] 
-                    },
-                    "author-only": {
-                        "type": [
-                            "string",
-                            "number",
-                            "boolean" 
-                        ] 
-                    },
-                    "uris": {
-                        "type": "array",
-                        "items": {
-                            "type": "string" 
-                        } 
-                    } 
-                },
-                "additionalProperties" : false 
-            } 
+                "$ref": "#/definitions/citationItemDefinition"
+            }
         },
         "properties": {
             "type": "object",
             "properties": {
                 "noteIndex": {
-                    "type": "number" 
-                } 
+                    "type": "number"
+                }
             },
-            "additionalProperties" : false 
-        } 
+            "additionalProperties": false
+        }
     },
-    "additionalProperties" : false 
+    "required": ["schema", "citationID", "citationItems"],
```

Comment:
Changed this to include `citationItems` because it should always return an array, even if empty IMHO.